src/internal/m365/service/onedrive/consts/consts.go

```diff
@@ -12,6 +12,10 @@ const (
 	RootPathDir = "root:"
 	// root id for drive items
 	RootID = "root"
+
+	// JWTQueryParam represents the query param embed in graph download URLs.
+	// It holds the JWT token.
+	JWTQueryParam = "tempauth"
```

Comment:
since this is used for all services that are backed by sharepoint/onedrive it feels like it would fit a bit better in the file it's used in (collection/drive/item.go)